core/src/main/scala/kafka/server/RaftRequestHandler.scala

```diff
@@ -48,6 +56,58 @@ class RaftRequestHandler(networkChannel: KafkaNetworkChannel,
             requestBody,
             response => sendResponse(request, Some(response)))
 
+        case ApiKeys.API_VERSIONS =>
+          sendResponse(request, Option(ApiVersionsResponse.apiVersionsResponse(0, 2)))
+
+        case ApiKeys.METADATA =>
+          val topics = new MetadataResponseData.MetadataResponseTopicCollection
```

Comment:
nit: verify that the metadata request only asked for the meta log topic and no others?